package.json

```diff
@@ -37,7 +37,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "prettier" : "npx prettier --write ."
```

Comment:
This would prettify all the file in the codebase while I would rather have done `npx prettier --write ./src`. Still good choice to add it as a command 👍